Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -70,6 +70,21 @@ init python:
             #Now we set the hwnd of this temporarily
             tip.hwnd = None
 
+    elif renpy.linux:
+        try:
+            import subprocess
+            subprocess.call(['notify-send', '--version'])
+        except OSError as e:
+            if e.errno == os.errno.ENOENT:
+                #Command wasn't found
+                store.mas_windowreacts.can_show_notifs = False
+                store.mas_utils.writelog(
+                    "[WARNING]: notify-send not found, disabling notifications.\n"
+                )
+            else:
+                #Some other error happened, what was it?
+                raise e
```

Comment:
not having `notify-send` isn't worthy of crashing the game. any sort of OSError exception here should set the `can_show_notifs` to False, and if its a different errno then just log the exception.

Reply:
Fixed, thanks.